keycloak/themes/hmda/login/register.ftl

```diff
@@ -96,4 +96,104 @@
+                        return { search: term }
+
+                        // Search based on "email" form field
+                        //var domain = emailToDomain($("#email").val());
+                        //return { domain: domain }
+                    },
+                    results: function(data, page) {
+                        return {
+                            // Each result MUST have an `id` attribute
+                            results: data.results 
+                        }
+                    }
+                },
+                escapeMarkup: function(markup) {
+                    return markup;
+                },
+                formatSelection: function(institution) {
+                    return  institution.name + ' (' + institution.id + ') ' + getStatusIcon(getFormEmail(), institution.domain[0]);
+                },
+                formatResult: function(institution) {
+
+                    return '<div class="container-fluid">' +
+                           '  <div class="row">' +
+                           '    <div styles="vertical-align:middle;" class="col-md-1">' +
+                           '      <h1>' + getStatusIcon(getFormEmail(), institution.domain[0]) + '</h1>' +
+                           '    </div>' +
+                           '    <div class="col-md-11">' +
+                           '      <div class="row">' +
+
+                           '        <div class="col-md-6">' +
+                           '          <div class="row">' +
+                           '            <div class="col-md-12">' +
+                           '              <h4>' + institution.name + '</h4>' +
+                           '            </div>' +
+                           '          </div>' +
+
+                           '          <div class="row">' +
+                           '            <div class="col-md-4">' +
+                           '               <i class="fa fa-gavel" aria-hidden="true"></i> ' + institution.regulator +
+                           '            </div>' +
+                           '            <div class="col-md-8">' +
+                           '               <i class="fa fa-envelope" aria-hidden="true"></i> ' + institution.domain[0] +
+                           '            </div>' +
+                           '          </div>' +
+
+                           '        </div>' +
+
+                           '        <div class="col-md-6">' +
+                           '          <div class="row"><strong>Respondent ID:</strong> ' + institution.id + '</div>' +
+                           '          <div class="row"><strong>EIN:</strong> 12-3456789</div>' +
+                           '          <div class="row"><strong>FDIC Charter:</strong> 999999</div>' +
+                           '          </div>' +
+                           '        </div>' +
+
+                           '      </div>' +
+                           '    </div>' +
```

Comment:
If ever there was an HTML string crying out to be templated ;)

I don't really want this to be changed (especially because I'd prefer not to have a front-end build step for the keycloak themes), but I will not fail to marvel at it.